jsonnet/kube-prometheus/components/kube-state-metrics.libsonnet

```
@@ -118,6 +118,8 @@ function(params) (import 'github.com/kubernetes/kube-state-metrics/jsonnet/kube-
     image: ksm._config.kubeRbacProxyImage,
   }),
 
+  // The override adding 'allowPrivilegeEscalation: false' can be deleted when
```

```suggestion
  // FIXME(ArthurSens): The override adding 'allowPrivilegeEscalation: false' can be deleted when
```

nit: Let's just add `FIXME` so it is easier to spot later ;)